language-examples/remedial-js/9.0-functional-programming/app/filter.js

```diff
@@ -0,0 +1,16 @@
+function filterExample01 () {
+  const sequence = [1,2,3,4,5,6,7]
+  const odds = sequence.filter((val) => val % 2 !== 0)
+
+  console.log(odds) // [1,3,5,7]
+}
+
+function filterExample02 () {
+  const buddies = [{ name: 'peter', age: 18 }, { name: 'sarah', age: 22 }, { name: 'gordon', age: 31 }, { name: 'jessica', age: 14 }]
```

Comment:
Might be nice to break these out onto their own lines in this file where we don't have to worry about vertical space as much.